src/pydap/net.py

```diff
@@ -20,3 +23,46 @@ def raise_for_status(response):
             headers=response.headers,
             comment=response.body
         )
+
+
+def follow_redirect(url, application=None, session=None,
```

Comment:
Could you add some exception catching around the cookie mangling in this function? I'm not completely sure of what the guarantees are that we can expect from `session.cookies.get_dict()` (i.e. *maybe* we can rely on `requests` handling). But I would hate for a client's `GET()` call to completely fall over because we made an incorrect string processing assumption.

Or... is it possible to simplify this, but not doing any dict->string conversion on the cookies? I don't think that actually using any dict/cookie_jar features (i.e. accessing by key name), so is it sufficient to just pass `req.headers['Cookie']` through the recursive calls?